api/src/user/dto/query.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { IsEmail, IsInt, IsOptional, IsString } from 'class-validator';
+import { Type } from 'class-transformer';
+
+export class QueryDto {
+  @IsString()
+  @IsOptional()
+  id?: string;
+
+  @IsString()
+  @IsOptional()
+  username?: string;
+
+  @IsEmail()
+  @IsOptional()
+  email?: string;
+
+  @IsInt()
+  @Type(() => Number)
+  @IsOptional()
+  page = 10;
+
+  @IsInt()
+  @Type(() => Number)
+  @IsOptional()
+  page_number = 0;
```

Comment:
An underscore, really? A man of your talents?

Reply:
This is because it defines the payload sent front he FE, and I don't think we want to `pageNumber` in a query string,